consensus-types/types/payload.go

```diff
@@ -600,7 +598,7 @@ func (p *ExecutionPayload) ToHeader() (*ExecutionPayloadHeader, error) {
 			WithdrawalsRoot:  p.GetWithdrawals().HashTreeRoot(),
 			BlobGasUsed:      p.GetBlobGasUsed(),
 			ExcessBlobGas:    p.GetExcessBlobGas(),
-			forkVersion:      p.forkVersion,
+			forkVersion:      p.GetForkVersion(),
```

Comment:
note: ExecutionPayloadHeader should then also be changed to have a `Versionable` instead of private `forkVersion`. But needs consideration as is constructed in many spots